### DISCUSSION TITLE
[@types/node] CompressionStream missing

### DISCUSSION THREAD

**Discussion by wmgries** [DefinitelyTyped/DefinitelyTyped#64322] [Issues with a @types package]
Looks like [CompressionStream](https://nodejs.org/api/webstreams.html#class-compressionstream) is missing from the `./types/node/stream/web.d.ts` file. 

The Node.js documentation indicates the following:

| Version | Changes |
|-|-|
| v18.0.0 | This class is now exposed on the global object. |
| v17.0.0 | Added in: v17.0.0 |

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
https://github.com/nodejs/node/pull/42225

**Comment by ghost** [comment]
Added in PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68214
